src/woo_publications/publications/api/serializers.py

```diff
@@ -498,6 +498,14 @@ class PublicationSerializer(serializers.ModelSerializer[Publication]):
             "otherwise an empty string is returned."
         ),
     )
+    url_publicatie_extern = serializers.SerializerMethodField(
+        label=_("external publication url"),
+        help_text=_(
+            "URL to the UI of the external application where the publication life "
+            "cycle is managed. Requires the global configuration parameter to be set, "
+            "otherwise an empty string is returned."
```

Comment:
> Requires the global configuration parameter to be set, otherwise an empty string is returned.

Which parameter is that? Or is that obvious to consumers of the API? As a noob, I only see that it's the same text as `url_publicatie_intern` :)

Reply:
In the admin there is a panel for global configs. This is about the nearly added `gpp_burgerportaal_publication_url_template`. The code is a carbon copy of Sergei's code for the other frontend component.

Reply:
:+1: I'll merge it. @MarcoKlerks can file a documentation ticket if this needs improvement.